Index: BankingApplication/src/main/java/com/ark/bankingapplication/views/StartUp.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- BankingApplication/src/main/java/com/ark/bankingapplication/views/StartUp.fxml	(revision 0c2715d51996b5fe861b00a38a30d6e06fec27fd)
+++ BankingApplication/src/main/java/com/ark/bankingapplication/views/StartUp.fxml	(date 1526140299662)
@@ -11,7 +11,7 @@
 <?import javafx.scene.layout.AnchorPane?>
 <?import javafx.scene.text.Font?>
 
-<AnchorPane fx:id="startAnchorPane" id="startAnchorPane" prefHeight="640.0" prefWidth="1080.0" stylesheets="@style.css" xmlns="http://javafx.com/javafx/9" xmlns:fx="http://javafx.com/fxml/1" fx:controller="main.java.com.ark.bankingapplication.views.StartUp">
+<AnchorPane fx:id="startAnchorPane" id="startAnchorPane" prefHeight="640.0" prefWidth="1080.0" stylesheets="@style.css" xmlns="http://javafx.com/javafx/9" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.ark.bankingapplication.views.StartUp">
    <children>
       <Label layoutX="57.0" layoutY="86.0" prefHeight="125.0" prefWidth="461.0" text="A.R.K. Banking" textFill="#806105">
          <font>
Index: BankingApplication/src/main/java/com/ark/bankingapplication/views/StartUp.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- BankingApplication/src/main/java/com/ark/bankingapplication/views/StartUp.java	(revision 0c2715d51996b5fe861b00a38a30d6e06fec27fd)
+++ BankingApplication/src/main/java/com/ark/bankingapplication/views/StartUp.java	(date 1526140345477)
@@ -3,7 +3,7 @@
  * To change this template file, choose Tools | Templates
  * and open the template in the editor.
  */
-package main.java.com.ark.bankingapplication.views;
+package com.ark.bankingapplication.views;
 
 import java.io.IOException;
 import java.net.URL;
